sdk/tables/data-tables/CHANGELOG.md

```diff
@@ -1,12 +1,10 @@
 # Release History
 
-## 13.1.2 (Unreleased)
+## 13.1.2 (2022-06-07)
```

Comment:
nit: We want to release core packages before non-core ones to have some room in case of issues. Should we release on Monday?